src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import lombok.Data;
+
+@Data
```

Comment:
If you have lazy fields, don't forget to look [this article](https://habr.com/ru/companies/haulmont/articles/564682/)